html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <label for="email">Email
```

Comment:
The `for` attribute pairs with the `id` attribute on its inputs. Please add an `id` to each of your inputs to match the value of the `for` attribute on its label 👍